dags/multipod/configs/maxtext_gce_config.py

```diff
@@ -29,27 +29,31 @@ def get_maxtext_nightly_config(
     tpu_cores: int,
     tpu_zone: str,
     time_out_in_min: int,
+    test_name: str,
     project_name: str = PROJECT_NAME,
     runtime_version: str = RUNTIME_IMAGE,
     network: str = "default",
     subnetwork: str = "default",
     is_tpu_reserved: bool = True,
+    num_slices: int = 1,
 ) -> task.TpuQueuedResourceTask:
   job_gcp_config = gcp_config.GCPConfig(
       project_name=project_name,
       zone=tpu_zone,
       dataset_name=metric_config.DatasetOption.XLML_DATASET,
   )
   current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
-  run_name = f"1slice-{tpu_version}_{tpu_cores}-maxtext-nightly-{current_datetime}"
+  run_name = (
+      f"{num_slices}slice-{tpu_version}_{tpu_cores}-maxtext-nightly-{current_datetime}"
+  )
 
   set_up_cmds = common.download_maxtext()
   run_model_cmds = (
       (
           "cd /tmp/maxtext && bash setup.sh MODE=nightly;"
           f' JAX_PLATFORM_NAME=TPU XLA_FLAGS="--xla_dump_to=/tmp/xla_dump/" RUN_NAME="{run_name}" &&'
           " python3 MaxText/train.py MaxText/configs/base.yml run_name=$RUN_NAME"
-          " base_output_directory=gs://tonyjohnchen-maxtext-nightly/"
+          " base_output_directory=gs://airflow-automated-maxtext-nightly-test/"
```

Comment:
Do you want to add your team's path here and re-use in the future? https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/dags/gcs_bucket.py#L24

Reply:
Done.